guide/designing-products/user-research.md

```diff
@@ -84,21 +83,24 @@ There are many ways in which research and development are similar to other digit
    layout = "float-right-desktop"
 %}
 
-Most digital products start with a specific country, culture, language or legal framework in mind. This is especially true for financial applications, which are typically highly regulated and need to slowly and strategically evolve. While Bitcoin did start with a single language, its use case was not tied to any particular context, and this has resulted in a much more diverse user base from the start.
+When designers begin to develop a product, they often start with a specific country, culture, language, or legal framework in mind. This is especially true for financial applications, which are highly regulated and need to evolve slowly and strategically. While Bitcoin was originally developed for an English speaking audience, it didn’t take long for its intended use case to naturally proliferate it.
 
-This has resulted in diverse and lively online communities that are deeply invested in Bitcoin. Designers and researchers have vast amounts of public conversation at their hands that help them better understand user needs and issues. These communities are also great for direct interaction to brainstorm ideas, get feedback on potential solutions, and get users involved in the design process.
+Online communities rapidly formed and multiplied around Bitcoin. Vast amounts of public conversation helped designers and developers comprehend what was possible, and what was needed from Bitcoin.
+
+Some of these links have been around for a while. The vast archives that they lead to still provide enormous value to anyone who wants to build Bitcoin products.
 
-Some of the following links have been around for a long time and provide a huge archive and history of information and conversation around bitcoin:
 - [Bitcoin on Github](https://github.com/bitcoin)
 - [Bitcoin mailing list](https://lists.linuxfoundation.org/mailman/listinfo/bitcoin-dev)
 - [Bitcointalk](https://bitcointalk.org/)
+- [Bitcoin subreddit](https://www.reddit.com/r/Bitcoin/)
+- [Bitcoin Core dev meeting logs](http://www.erisian.com.au/meetbot/bitcoin-core-dev/)
 - [Bitcoin subreddit](https://www.reddit.com/r/bitcoin)
 
 </div>
 
 ### An unknowable userbase
 
-The Bitcoin protocol works with addresses and has otherwise no sense of identity that allows us to find out who uses the system and performs various functions. Analyzing the blockchain allows for some basic transaction behavior analysis, but analysis cannot tie it to specific persons or locations. While this is generally a fantastic user benefit for privacy and security, it is harder to understand user behavior and recruit users for research.
+Bitcoin only knows its users through pseudonymous addresses. We can analyze on-chain metrics for things like fundamental transaction behaviors, but analysis of addresses and transactions beyond this can be complicated. While Bitcoin's privacy provides is a fundamental aspect, it can make things challenging for user researchers.
```

Comment:
```suggestion
Bitcoin only knows its users through pseudonymous addresses. We can analyze on-chain metrics for things like fundamental transaction behaviors, but analysis of addresses and transactions beyond this can be complicated. While the privacy Bitcoin provides is a fundamental aspect, it can make things challenging for user researchers.
```
Fixed the sentence. Still reads a bit funny though.